tests/accel/upgrade/upgrade.go

```diff
@@ -0,0 +1,172 @@
+package upgrade
```

Comment:
Please move it to ./tests/accel/upgrade/tests directory

Reply:
Done, thanks